paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -877,7 +906,16 @@ class CustomerSheetViewModelTest {
             customerAdapter = FakeCustomerAdapter(
                 canCreateSetupIntents = false,
                 onSetupIntentClientSecretForCustomerAttach = null,
-                onAttachPaymentMethod = { Result.failure(Exception("Cannot attach payment method")) },
+                onAttachPaymentMethod = {
+                    CustomerAdapterResult.Failure(
+                        cause = APIException(
+                            stripeError = StripeError(
+                                message = "Cannot attach payment method."
+                            )
+                        ),
+                        displayMessage = "Cannot attach payment method."
```

Comment:
Same

Reply:
Ping. I think there's quite a few of these. Can we make sure we're testing things in a way that's not possible to grab the message from the wrong place.